### PR TITLE
Fortresses and sparta wonder

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/spart/wonder.xml [hunk 1]
```diff
@@ -7,6 +7,11 @@
   <Identity>
     <Civ>spart</Civ>
     <SpecificName>Hieron tēs Artemidos Orthias</SpecificName>
+    <History>
+        Hieron tēs Artemidos Orthias or the Sanctuary of Artemis Orthias was a place where many religious festivals were held, including a coming of age rite where Spartan boys would try to steal some boys would try to steal cheese from the altar, and other boys would try to catch them and beat them.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/spart/wonder.xml:11]:
```suggestion
        Hieron tēs Artemidos Orthias, or the Sanctuary of Artemis Orthias, was a place where many religious festivals were held, including a coming-of-age rite where some boys would try to steal cheese from the altar, and other boys would try to catch them and beat them.
```

File: 0-ad-history-mod-package/simulation/templates/structures/spart/wonder.xml [hunk 2]
```diff
@@ -7,6 +7,11 @@
...
+        Archaeological evidence is scarce, but it is thought that originally this site was dedicated to a goddess named Orthias, who seemed to be a fertility goddess who watched over children. Over time however, her character was closer and closer associated to the Greek Godess Artemis, and later inscriptions read Artemis Orthias before eventually just reading Artemis.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/spart/wonder.xml:12]:
```suggestion
        Archaeological evidence is scarce, but it is thought that originally this site was dedicated to a goddess named Orthias, who seemed to be a fertility goddess who watched over children. Over time, however, her character was closer and closer associated to the Greek godess Artemis, and later inscriptions read Artemis Orthias before eventually just reading Artemis.
```

File: 0-ad-history-mod-package/simulation/templates/structures/spart/wonder.xml [hunk 3]
```diff
@@ -7,6 +7,11 @@
...
+        The sanctuary had been destroyed and rebuilt over the years, but its use was continued to into the fourth century AD.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/spart/wonder.xml:13]:
```suggestion
        The sanctuary had been destroyed and rebuilt over the years, but its use was continued into the fourth century AD.
```

File: 0-ad-history-mod-package/simulation/templates/units/han/siege_ram.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Entity parent="encyclopedia_siege_ram|encyclopedia_siege_ram|template_unit_siege_ram">
+<Entity parent="encyclopedia/encyclopedia_siege_ram|encyclopedia/encyclopedia_siege_ram|template_unit_siege_ram">
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/units/han/siege_ram.xml:2]:
```suggestion
<Entity parent="encyclopedia/encyclopedia_siege_ram|template_unit_siege_ram">
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is this clear to be merged?

**Comment by indoptogopt** [comment]
Have you not commited the one suggestion on purpose?

**Comment by TheShadowOfHassen** [comment]
Oops didn't see it. Clear now?

**Comment by indoptogopt** [comment]
Yes

**Pull request merged**